l1-contracts/contracts/bridge/ntv/NativeTokenVault.sol

```diff
@@ -208,7 +204,17 @@ abstract contract NativeTokenVault is INativeTokenVault, IAssetHandler, Ownable2
             receiver: _receiver,
             amount: _amount
         });
-        _bridgeMintData = _data;
+        bytes memory erc20Metadata;
+        {
+            erc20Metadata = getERC20Getters(bridgedToken, true, originChainId[_assetId]);
```

Comment:
let's add a comment that explains why `originChainId` is non-zero here

Reply:
maybe even better, we could have an explicit error if either the address of the origin chain id is 0